### DISCUSSION TITLE
How to debug Apollo Server Files in VSCode?

### DISCUSSION THREAD

**Discussion by hazem-alabiad** [apollographql/apollo-server#5425] [Q&A]
How can I debug a file in Apollo Server Project e.g. "federations.js" using the debugger in VSCode?

**Comment by glasser** [comment]
I don't think there's anything particularly strange about Apollo Server that makes it any easier or harder to debug than any other Node server? https://code.visualstudio.com/docs/nodejs/nodejs-debugging is the main docs on doing that.
